spec/features/users/sign_in_spec.rb

```diff
@@ -116,30 +116,16 @@
 
     context 'javascript enabled', js: true do
       it 'pops up session timeout warning' do
-        # session of length 0.2.minutes == 12 sec
-        # we need session timeout long enough to allow session to live
-        # between requests (esp at travis), but short enough so that
-        # using_wait_time does not expire during find_link().
-        session_ttl = 0.2.minutes # 12 sec
-        page_wait_time = 15 # seconds, must be >= session_ttl
+        session_ttl = 0.01.minutes # 0.6 sec
+        page_wait_time = 1 # seconds, must be >= session_ttl
 
         allow(Devise).to receive(:timeout_in).and_return(session_ttl)
 
-        user = sign_in_and_2fa_user
-        click_link(t('links.sign_out'))
+        visit root_path
 
         Capybara.using_wait_time(page_wait_time) do
-          find_link(t('forms.buttons.continue')).trigger('click')
+          expect(page).to have_link(t('forms.buttons.continue'))
```

Comment:
Actually, it looks like we already had both cases covered here: https://github.com/18F/identity-idp/blob/master/spec/features/users/sign_in_spec.rb#L91-L106

Do we even need this test here?